tests/sentry/tsdb/test_redissnuba.py

```diff
@@ -27,12 +27,12 @@ def test_redissnuba_connects_to_correct_backend():
 
     for method in methods:
         for model in should_resolve_to_redis:
-            assert "redis" == selector_func(method, get_callargs(model))
+            assert "redis" == selector_func(method, get_callargs(model), None)
```

Comment:
Can revert all changes to this file then I guess?

Reply:
oh right, I thought your suggested change went the other way